### ISSUE TITLE
youtube stopped working

### ISSUE THREAD

**Issue by static-void** [ColumPaget/Movgrab#5]
Me again! I'm afraid youtube stopped working, unless I am doing something stupid... I have built from master with ssl and largefiles enabled (on Linux). Thank you...

**Comment by ColumPaget** [comment]
Yes, youtube has stopped working, and to be honest I think I'm going to abandon movgrab. I can't keep up with the changes youtube keeps making, and I see it's not just me, other tools, like smtube, are having a lot of trouble too.

Also, youtube is clearly dying. The in-browser user experience is laughably bad, with the site constantly butting in and forcing me to watch, say, adverts about funeral products while I'm trying to watch movie reviews. Yeah, I really want to use a site that reminds of my own mortality during my leisuretime. movgrab and youtube-dl would help with that, but youtube wants to lock those out to barrage us with gauche ads. Additionally I note I see the same adverts over and over, and these are often 'low class' adverts that can't be bringing in much revenue, which implies the platform isn't attracting high-class advertisers. Content creators cannot even speak plainly anymore for fear of being demonitized, having to use complex euphemisms to avoid an ever growing list of topics youtube considers verboten. Most of the content creators I follow complain constantly about the site, and a few of them are starting to jump ship, a trend that will only accelerate going forwards.

Youtube just isn't as important to me as it once was. Some of my favorite channels, like Issac Arthur, are now available elsewhere, and the workload to keep movgrab working and keep figuring out what youtube have done to break it this time, just isn't worth the declining payout I get from the site.

**Comment by static-void** [comment]
That is a shame, because movgrab is great for low resource machines where python does not run well (like the zipit z2). But I understand your reasons. Thanks

**Comment by barracuda156** [comment]
@ColumPaget There is a use-case for YouTube – music videos (and tbh I do not know where to get those otherwise, aside from torrents, but even there not all will be available easily).

Besides, does `Movgrab` still work with [some] other sites and the current `libUseful`?
